libraries/bezier-rs/src/bezier/core.rs

```diff
@@ -141,13 +142,34 @@ impl Bezier {
 		}
 	}
 
-	/// Convert `Bezier` to SVG `path`.
-	pub fn to_svg(&self) -> String {
+	/// Returns an SVG representation of the `Bezier`.
+	pub fn to_svg(&self, options: ToSVGOptions) -> String {
+		let anchor_arguments = options.formatted_anchor_arguments();
+		let anchor_circles = format!(
+			r#"<circle cx="{}" cy="{}" {anchor_arguments}/><circle cx="{}" cy="{}" {anchor_arguments}/>"#,
+			self.start.x, self.start.y, self.end.x, self.end.y
+		);
+
+		let handle_point_arguments = options.formatted_handle_point_arguments();
+		let handle_circles = match self.handles {
+			BezierHandles::Linear => None,
+			BezierHandles::Quadratic { handle } => Some(format!(r#"<circle cx="{}" cy="{}" {handle_point_arguments}/>"#, handle.x, handle.y)),
+			BezierHandles::Cubic { handle_start, handle_end } => Some(format!(
+				r#"<circle cx="{}" cy="{}" {handle_point_arguments}/><circle cx="{}" cy="{}" {handle_point_arguments}/>"#,
+				handle_start.x, handle_start.y, handle_end.x, handle_end.y
+			)),
+		};
+
 		format!(
-			r#"<path d="{SVG_ARG_MOVE}{} {} {}" stroke="black" fill="none"/>"#,
+			r#"<path d="{SVG_ARG_MOVE}{} {} {}" {}/><path d="{}" {}/>{}{}"#,
 			self.start.x,
 			self.start.y,
-			self.svg_curve_argument()
+			self.svg_curve_argument(),
+			options.formatted_curve_arguments(),
+			self.svg_handle_line_argument().unwrap_or_else(|| "".to_string()),
+			options.formatted_handle_line_arguments(),
+			anchor_circles,
+			handle_circles.unwrap_or_else(|| "".to_string()),
```

Comment:
Perhaps this function could take a mutable string and use `write!` macro to avoid excess heap allocations (like formatting anchor_circles` and `handle_circles` and then formatting them again)?

Reply:
I've made what I think is the change you requested - could you confirm that I did what you expected?